packages/NodeTypeResolver/NodeVisitor/FunctionLikeParamArgPositionNodeVisitor.php

```diff
@@ -15,9 +15,6 @@
 
 final class FunctionLikeParamArgPositionNodeVisitor extends NodeVisitorAbstract
 {
-    /**
-     * @return Node
-     */
```

Comment:
I understand what you mean, but how do you propose to generalize this? Sometimes this kind of narrowing might be intentional, like here, but other times it might be a mistake. I think that using `@phpstan-return` instead of just `@return`  could help. Same goes for `@var` and `@param` in the sibling rules, phpstan prefixed variants should be ignored..